server/models/Staff.php

```diff
@@ -41,14 +41,18 @@ public function getDefaultProps() {
         return [
             'level' => 1,
             'ownStatList' => new DataStoreList(),
-            'sendEmailOnNewTicket' => 0 
+            'sendEmailOnNewTicket' => 0
         ];
     }
 
     public static function getUser($value, $property = 'id') {
         return parent::getDataStore($value, $property);
     }
 
+    public function canManageTicket(Ticket $ticket){
```

Comment:
We should a function with the same name in User but it should check it is owner

```
    public function canManageTicket(Ticket $ticket){
        return $ticket->isOwner($this);
    }
```
Then we can remove the isOwner checks from the other if's. Please make tests before that change.